tsdb/tsdbutil/chunks.go

```diff
@@ -65,24 +64,27 @@ func ChunkFromSamplesGeneric(s Samples) (chunks.Meta, error) {
 	}
 
 	ca, _ := c.Appender()
+	var newc chunkenc.Chunk
```

Comment:
Strictly speaking, this should be at least `newC`. But since you used `emptyChunk` above, I would be as generous here and call this `newChunk`.